backend/src/main/java/com/handong/rebon/shop/application/CategoryService.java

```diff
@@ -0,0 +1,58 @@
+package com.handong.rebon.shop.application;
+
+import com.handong.rebon.exception.category.CategoryExistException;
+import com.handong.rebon.exception.category.CategoryNoParentException;
+import com.handong.rebon.shop.application.dto.CategoryRequestDto;
+import com.handong.rebon.shop.domain.category.Categories;
+import com.handong.rebon.shop.domain.category.Category;
+import com.handong.rebon.shop.domain.repository.CategoryRepository;
+
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@Service
+public class CategoryService {
+
+    private final CategoryRepository categoryRepository;
+
+    public Long create(String name) {
+        isCategoryExist(name);
+
+        Category newCategory = Category.builder()
+                                       .name(name)
+                                       .children(new Categories())
+                                       .build();
+        categoryRepository.save(newCategory);
+        return newCategory.getId();
```

Comment:
이렇게 해줘도 되겠지만 명시적으로 반환값을 사용하는건 어떻게 생각하시나요?
```suggestion
        Category savedCategory = categoryRepository.save(newCategory);
        return savedCategory.getId();
```

Reply:
아! 명시적으로 하는게 좋은것 같습니다 반영한줄알았는데 안되어있군요 수정하겠습니다 ㅎㅎ